ui/app/services/version.js

```diff
@@ -63,8 +63,8 @@ export default class VersionService extends Service {
   @task({ drop: true })
   *getVersion() {
     if (this.version) return;
-    // Fetch seal status with token to get version
-    const response = yield this.store.adapterFor('cluster').sealStatus(false);
+    // Fetch seal status to get version
+    const response = yield this.store.adapterFor('cluster').sealStatus();
```

Comment:
👉 this is the fix -- this is what ensures the UI hides the version by calling `/seal-status/` unauthenticated. 

@chelshaw i saw that this change was recent and intentionally added in https://github.com/hashicorp/vault/pull/25998, so i wanted to check in with you about this as a solution. because we're just trying to grab the `version` property from the response, i wondered if calling the endpoint unauthenticated in this specific instance would be okay? it seems like if we needed more information that's only returned when the call is authenticated that we could just make a separate call then. what do you think?